tests/common/nebula_service.py

```diff
@@ -130,6 +130,8 @@ def start(self, debug_log=True):
                                                   metad_ports[0],
                                                   ports,
                                                   debug_log)
+            if server_name == 'graphd':
+                command += ' --enable_authorize=true'
```

Comment:
please append this flag in `_format_nebula_command` function.

Reply:
Fixed.